### PR TITLE
remove docfx check

### PR DESCRIPTION
**Pull request by puckpuck**
docfx is failing on relative links to folders. We require these types of links for our service docs.

Per the SIG meeting on 9/26, we determined that the docfx checks were providing no value-add and should be removed.  This PR comments out the check (we can remove it completely on a subsequent PR)

### CODE REVIEW

**Review by reyang — APPROVED** [review]

**Review by fatsheep9146 — APPROVED** [review]

**Review by mviitane — APPROVED** [review]

**Comment by cartersocha** [comment]
@puckpuck we might need to remove something in the repo because the docfx check is blocking this pr for not running

**Comment by reyang** [comment]
> @puckpuck we might need to remove something in the repo because the docfx check is blocking this pr for not running

Maintainers need to remove this from the branch policy:

image

**Comment by cartersocha** [comment]
@austinlparker youll need to remove the rule. I don’t have permissions to make repo changes like that

**Comment by austinlparker** [comment]
I don't have access to branch protection rules; Let me open an issue to have maintainers get admin access.

**Comment by reyang** [comment]
> I don't have access to branch protection rules; Let me open an issue to have maintainers get admin access.

All maintainers on this project should have access now.

**Comment by puckpuck** [comment]
I removed the branch protection rule that @cartersocha posted a screenshot about, so will merge this now

**Pull request merged**
